### PR TITLE
[12.0][FIX] openupgrade_records: several fixes

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
This PR:
-  solves https://github.com/OCA/OpenUpgrade/issues/1624
- reports changes like https://github.com/OCA/OpenUpgrade/pull/1674#pullrequestreview-202486667
- and other fixes

The minor fix commits will be squashed before merge.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
Looking good! Can you provide a branch that contains the analysis with this change so that we can validate the results? You may need to run the analysis against the older version with this change backported, because you changed the output format.

File: odoo/addons/openupgrade_records/lib/compare.py [hunk 1]
```diff
@@ -95,35 +95,42 @@ def report_generic(new, old, attrs, reprs):
                 text = "now required"
                 if new['req_default']:
                     text += ', default = %s' % new['req_default']
-                fieldprint(old, new, None, text, reprs)
+                fieldprint(old, new, '', text, reprs)
+        elif attr == 'stored':
+            if old[attr] != new[attr]:
+                if new['stored']:
+                    text = "now is stored"
```
Comment (StefanRijnhart) [inline odoo/addons/openupgrade_records/lib/compare.py]: Slight preference for "now stored" (or else "is now stored", but not "now is stored").

**Comment by StefanRijnhart** [comment]
Great work!

**Review by pedrobaeza — APPROVED** [review]
Indeed congratulations for the improvements. There's still one missing feature that I would love to have, that is to report added and missing models in the module itself, but this should been done counting references and seeing if it's the last/first one, and that's for another PR of course. Merging this one.

**Pull request merged**